Sources/SwiftDocC/CMakeLists.txt

```diff
@@ -0,0 +1,458 @@
+#[[
+This source file is part of the swift-format open source project
+
+Copyright (c) 2024 Apple Inc. and the DocC project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See https://swift.org/LICENSE.txt for license information
+#]]
+
+add_library(SwiftDocC
```

Comment:
Does these lists need to be maintained whenever we add a new file to either of these targets?

Reply:
Yes, that is the general pattern for CMake - you manage the source list explicitly so that the build system knows what is part of the build and what is not.

Reply:
Is there any way to specify directories instead of files lists? I'm concerned about the maintenance burden of keeping this list up to date.

Reply:
No, not without basically making the build system brittle and unreliable. CI would easily catch any files removed. Any file that the build doesn't catch where the list is not maintained would imply that it can be dropped because there is no reference to the types/functions it defines.